network/preview/cardano-node/config.json

```diff
@@ -4,22 +4,23 @@
   "ApplicationName": "cardano-sl",
   "ApplicationVersion": 0,
   "ByronGenesisFile": "../genesis/byron.json",
-  "ByronGenesisHash": "72593f260b66f26bef4fc50b38a8f24d3d3633ad2e854eaf73039eb9402706f1",
-  "EnableP2P": false,
+  "ByronGenesisHash": "83de1d7302569ad56cf9139a41e2e11346d4cb4a31c00142557b6ab3fa550761",
+  "EnableP2P": true,
```

Comment:
FYI: `preview` is the non-p2p version of the config and we have a `preview_p2p` variant in this repo. You copied the p2p version here (which might be the only working right now?)

Reply:
I copy/pasted what is available from [Cardano book](https://book.world.dev.cardano.org/environments.html#preview-testnet) but I have pretty much no idea of what I am doing here, except it works 🤷 
Feel free to improve or reject this P.R.